src/ctia/task/migration/migrations.clj

```diff
@@ -162,4 +254,6 @@
                  pluralize-target)
    :1.0.0 (comp (append-version "1.0.0")
                 (rename-observable-type "pki-serial" "pki_serial")
-                simplify-incident)})
+                simplify-incident)
+   :1.1.0 (comp (append-version "1.1.0")
```

Comment:
I was redacting the tenzin issue, and while filling the migration parameter I remembered to modify this
```suggestion
   :investigation-actions (comp (append-version "1.1.0")
```